test/about-self.js

```diff
@@ -122,6 +156,8 @@ test('should load about-self from disk', (t) => {
   })
 })
 
+
+
```

Comment:
Can you run `npm run format-code` before making a git commit? It should have run automatically as a git hook, but for some reason on your local dev env it didn't. That would get rid of these added empty lines.